scripts/installer.sh

```diff
@@ -513,7 +513,10 @@ main() {
 			;;
 		pacman)
 			set -x
-			$SUDO pacman -Sy
+			# https://github.com/tailscale/tailscale/issues/12186
+			if [ "$OS" != "arch" ]; then
+				$SUDO pacman -Sy
+			fi
```

Comment:
Sounds like the issue is not specific to Arch, but any distro using pacman.
We should probably never run `pacman -Sy`.

Reply:
Why does it need to use `pacman -Sy` anyway?

Reply:
It's similar to all other repos - refresh the index to fetch the latest version. Arch is the only one that actually discourages this practice.

It matters more on rpm and deb-based distros, where we own the repo and add it in this same installer script. Without a refresh, apt/dnf/yum wouldn't know about the tailscale package.

I agree that it is probably not necessary on pacman-based distros, unless you have a fresh install where `pacman -Sy` has never ran before (e.g. when you run a fresh `archlinux` docker container`).